content/operate/rc/databases/rdi/define.md

```diff
@@ -0,0 +1,68 @@
+hideListLinks: true
+weight: 2
+---
+
+After you have [prepared your source database]({{<relref "/operate/rc/databases/rdi/setup">}}) and connection information, you can set up your new pipeline. To do this:
+
+1. [Define the source connection](#define-source-connection) by entering all required source database information.
+2. [Define the data pipeline](#define-data-pipeline) by selecting the data that you want to sync from your source database to the target database.
+
+## Define source connection
+
+1. In the [Redis Cloud console](https://cloud.redis.io/), go to your target database and select the **Data Pipeline** tab.
+1. Select **Start pipeline setup**. 
+1. Enter a **Pipeline name**. This pipeline name will be the prefix to all keys generated by this pipeline in the target database.
+1. Enter the **Deployment CIDR** for your pipeline, or use the one generated for you. This CIDR should not conflict with your apps or other databases.
+1. In the **Connectivity** section, enter the **PrivateLink service name** of the [PrivateLink connected to your source database]({{< relref "/operate/rc/databases/rdi/setup#set-up-connectivity" >}}).
+1. Enter your database details. This depends on your database type, and includes:
+    - **Port**: The database's port
+    - **Database**: Your database's name, or the root database *(PostgreSQL, Oracle only)*, or a comma-separated list of one or more databases you want to connect to *(SQL Server only)*
+    - **Database Server ID**: Unique ID for the replication client. Leave as default if you don't use replication *(mySQL and mariaDB only)*
+    - **PDB**: Name of the Oracle pluggable database *(Oracle only)*
+1. Enter the ARN of your [database credentials secret]({{< relref "/operate/rc/databases/rdi/setup#share-source-database-credentials" >}}) in the **Source database secrets ARN** field.
+1. Select **Start pipeline setup**.
+1. Redis Cloud will attempt to connect to PrivateLink. Accept the incoming connection on AWS PrivateLink to proceed. See [Accept or Reject PrivateLink connection requests](https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html#accept-reject-connection-requests).
+
+    If Redis Cloud can't find your PrivateLink connection, make sure that the PrivateLink service name is correct and that Redis Cloud is listed as an Allowed Principal for your VPC. See [Set up connectivity]({{<relref "/operate/rc/databases/rdi/setup#set-up-connectivity">}}) for more info.
+
+At this point, Redis Cloud will provision the pipeline infrastructure that will allow you to define your data pipeline. 
+
+Pipelines are provisioned in the background. You aren't allowed to make changes to your data pipeline or to your database during provisioning. This process will take a long time, so you can close the window and come back later.
+
+When your pipeline is provisioned, select **Complete setup**. You will then [define your data pipeline]({{<relref "/operate/rc/databases/rdi/define">}}).
```

Comment:
This links to the top of the page. I think you probably want to link to the section below?
```suggestion
When your pipeline is provisioned, select **Complete setup**. You will then [define your data pipeline](#define-data-pipeline).
```